src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
-// write your code here
+let calculate1 = 0;
+let calculate2 = 0;
+let totalCalculate = '';
+const text = document.getElementsByClassName('population');
+const totalPopulation = document.querySelector('span.total-population');
+const averagePopulation = document.querySelector('span.average-population');
+
+for (let i = 0; i < text.length; i++) {
+  calculate1 += +text[i].textContent.split(',').join('');
+}
+
+calculate1 = calculate1 + '';
```

Comment:
Converting `calculate1` to a string immediately after calculating the sum may lead to unexpected results in subsequent calculations. Consider keeping it as a number until all calculations are complete.